src/slate/slate.h

```diff
@@ -54,16 +54,17 @@ typedef struct samwise_slate
     /*
      * Command switch
      */
-    queue_t task1_data; // queues of this kind will exist for each task called
-                        // from radio com
-    queue_t task2_data;
+    queue_t take_photo_task_data;
+    queue_t download_photo_task_data;
+    queue_t take_and_send_photo_task_data;
```

Comment:
What's the advantage of having separate task_data queues for payload commands (which all have the same data structure PAYLOAD_COMMAND_DATA)?